src/pages/Summary/Summary.data.tsx

```diff
@@ -120,9 +120,13 @@ function ChildrenSuccessInstitutionProfileUpdate(): JSX.Element {
     <>
       Please allow 24-48 hours for a response during normal business hours. If
       you need further assistance please{' '}
-      <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update your financial institution profile: Questions after submitting form'>
+      <Link
+        className='border-b-[1px]'
+        href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update your financial institution profile: Questions after submitting form'
```

Comment:
Great! Do we need this for this other `Link` on the `Summary` page as well?

https://github.com/cfpb/sbl-frontend/pull/354/commits/fab5a725fc4704aaa09696eb9f46e1e646d1d3e8#diff-245c110e80bc6d42c2292f783a58eab95844704c4b0457ec4592f860d64570a0R110

Reply:
@billhimmelsbach Thanks for catching that. Updated.